src/integrations/bluetooth-low-energy/bluetooth-low-energy.service.ts

```diff
@@ -291,12 +316,53 @@ export class BluetoothLowEnergyService
    * @param name - Name for the new device tracker
    * @returns Registered device tracker
    */
-  protected createDeviceTracker(id: string, name: string): DeviceTracker {
+  protected createDeviceTracker(
+    id: string,
+    name: string,
+    deviceCustomizations: deviceCustomizations
```

Comment:
I think the `deviceCustomizations` aren't needed here yet, as those would only be used for auto discovered entities - which Home Assistant does not support for device trackers yet.

Reply:
Yep. I went for consistency with the other entities knowing home-assistant module was filtering out the device-tracker discovery. Felt like a concern the ble side of things didn't need to worry about. But to be honest, there was probably a bit of wishful thinking that the HA guys will accept [this](https://github.com/home-assistant/core/pull/42327). I fear hacktoberfest may delay things on that !

Reply:
Ouh, nice! I've subscribed to that PR, would solve a great annoyance in the room-assistant setup. :)